os/include/tinyara/binary_manager.h

```diff
@@ -70,18 +70,38 @@
 /* Mount point for User binaries */
 #define BINARY_DIR_PATH                  CONFIG_MOUNT_POINT"bins"
 
+/* The length of binary file or kernel partition path.
+ * A path is same like below:
+ *  - User filepath : "/<mount_path>/<binary_name>_<binary_version>"
+ *         length : CONFIG_NAME_MAX + BIN_NAME_MAX + binary_version (32) + /,_(3) + '\0'(1)
+ *  - Kernel partpath : "/dev/mtdblock#" defined as BINMGR_DEVNAME_FMT
+ * So define binary path length as (CONFIG_NAME_MAX + BIN_NAME_MAX + 40) with some extra.
+ */
+#define BINARY_PATH_LEN                  (CONFIG_NAME_MAX + BIN_NAME_MAX + 40)
+
 /* Binary states used in state callback */
 enum binary_statecb_state_e {
 	BINARY_STARTED = 0,           /* Binary is started */
 	BINARY_UNLOADED = 1,          /* Binary is unloaded */
 	BINARY_READYTOUNLOAD = 2,     /* Binary will be unloaded */
 };
 
+enum binary_state_e {
+	BINARY_UNREGISTERED = 0,     /* File is not existing or Partition is not registered */
+	BINARY_INACTIVE = 1,         /* File is not existing or Partition is registered, but binary is not loaded yet */
```

Comment:
There is one situation (`File is not existing`), but two states(`BINARY_UNREGISTERED`, `BINARY_INACTIVE`).
I think this should be `File is existing`.

Reply:
There was a typo. I modified